### DISCUSSION TITLE
Simple JWT validation

### DISCUSSION THREAD

**Discussion by pelord** [OpenIDC/mod_auth_openidc#969] [Q&A]
I would like to know if this mod could simply validate a JWT token, provided by the Authorization header (Bearer ... ), based on issuer and signature (secret key) validation.

My JWT token is created by a custom login API.

**Comment by zandbelt** [comment]
that is possible but it is a deprecated feature, better use https://github.com/zmartzone/mod_oauth2 for that

**Reply by pelord** [reply]
I will test it.
